Extensions/video_downloader.css

```diff
@@ -0,0 +1,33 @@
+.xvd-button{
+  position:absolute;

```

Comment:
we don't lint the CSS but we prefer it to be in the same kind of style, i.e.
```css
.xvd-button {
    position: absolute;
```
etc.

Reply:
gotcha +